src/components/BulkEnrollmentPage/BulkEnrollment.scss

```diff
@@ -39,7 +43,12 @@
 }
 
 .sticky-stepper-footer {
+  height: 70vh;
```

Comment:
this is still a more dynamic unit of measure, but I do wonder about putting a specific size in here.

Reply:
this was out of necessity, here 😞  I was pair programming with Lael and we were totally unable to get the footer at the bottom of the page while having a scrollable table and this seemed to be the only solution we could find. If there is a better answer I'm more than happy to change.

Reply:
`vh` is somewhat dynamic, as it changes based on the height of the window. It's not 100% ideal, but I think it's OK?